docker-compose.yml

```diff
@@ -1,23 +1,27 @@
-version: '3.1'
+version: '2'
 
 services:
 
-  nameless:
-    build:
-      context: src
+  namelessmc:
+    image: derkades/namelessmc-nginx
+    ports: ['80:80']
+    volumes: ['/srv/namelessmc/web:/data'] # same as php container
+    # environment:
+    #   PHP_FPM: namelessmc_php:9000
+    depends_on: [namelessmc_php]
     restart: unless-stopped
-    links:
-      - db
-    ports:
-      - "80:80"
 
-  db:
-    image: mysql:5.7
+  namelessmc_php:
+    image: derkades/namelessmc-php
```

Comment:
And here

Reply:
The `derkades/` prefix is necessary to allow downloading the image from docker hub. I'll contact Samerton about creating a namelessmc organisation instead of using my personal account for this.

Reply:
Do you own https://hub.docker.com/u/namelessmc?

Reply:
Even better, change compose file to local build context since we have full source code required to build. We never need to grab a pre-built image remotely.

Reply:
And yes I have access to the Docker Hub organization

Reply:
This compose file was intended as an example for end users, but indeed a development file would be nice. I'll create one with local building.